### PR TITLE
Updated Japanese translation

### PR DESCRIPTION
**Pull request by 8h42**
Updated Japanese translation for Policy Reminder Popup

### CODE REVIEW

**Comment by oerms** [comment]
You referenced `cqui_InGameText_ja.xml` but there is no such file as I see it.

**Comment by 8h42** [comment]
@oerms
I forgot to add it.
Thank you for your comment.

File: Assets/Text/cqui_InGameText_ja.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (oerms) [inline Assets/Text/cqui_InGameText_ja.xml]: Indentation seems to be wrong here. I think when I copied the original file, I did not replace the tabs by 2 spaces.

Comment (Azurency) [reply Assets/Text/cqui_InGameText_ja.xml]: Yes, the indentation is supposed to be tow spaces, not tab with a size of two. In all the files.

**Comment by 8h42** [comment]
Still need any fix?

**Pull request merged**